docs/adr/001-record-architectural-decisions.md

```diff
@@ -0,0 +1,22 @@
+# ADR 001 - Record architectural decisions
+
+Date: 26-11-2024
+
+## Status
+
+Accepted
+
+## Context
+
+We need to record the architectural decisions made on this project. Architectural Decision Records as
+[described by Michael Nygard](https://cognitect.com/blog/2011/11/15/documenting-architecture-decisions) provides a
+framework for writing down these decisions.
+
```

Comment:
I think it would be more understandable if we give some context as to why we are starting the ADR now. Maybe we could write that the ADR has been started at a point where the project has been running for several years but this seemed to be the right time because the project has evovlved to become more complex and that there are more developers involved than previous. This calls for summarizing some of the baselines of the project so we make sure to keep some of the original thoughts and decisions in the project documented and so that we remember to note if we make any changes as we move forward.